verify/db_test.go

```diff
@@ -33,13 +34,6 @@ func TestDelegationsDB(t *testing.T) {
 			}},
 			initErr: ErrInvalidThreshold,
 		},
-		{
-			testName: "invalid keys",
-			delegations: &data.Delegations{Keys: map[string]*data.PublicKey{
-				"a": {Type: data.KeySchemeEd25519},
-			}},
-			initErr: ErrWrongID{},
```

Comment:
re-add this test to test non-sha key IDs and expect no error?

Reply:
Oof, it's a whole lot of work to expect no error because you need to have things signed and so on. I can get to a `ErrNoSignatures` with a lot less work, which *would have been* a `ErrWrongID` before. Is that okay?

Reply:
Ah, let's close, we have the test coverage from the other tests you had. thanks!

Reply:
Ah but you had added those! already, can you add a quick comment explaining that those tests are testing that we don't fail on non-sha key IDs just in case those tests break later?